### PR TITLE
fix: teleperformance scene

### LINKED ISSUE #1372: [QA] Multiplayer not working at Teleperformance due to Json parse
### **Explorer Alpha build version:**
v.0.10.0

### **Issue Description:**

On Teleperformance scene `/goto -96,100` the multiplayer isn't working since it's receiving a JSON file that cannot be parsed.

#### **Evidence:**

```
[JAVASCRIPT] (-96, 108): SceneError: Error receiving data: SyntaxError: Unexpected non-whitespace character after JSON at position 3 stackTrace:     at HostDelegate.<anonymous> (<anonymous>)
    at Object.error (Script [20]:45:45)
    at Script [18]:27:25
0x00007ff706d174dd (Unity) StackWalker::GetCurrentCallstack
0x00007ff706d1c5a9 (Unity) StackWalker::ShowCallstack
0x00007ff707cfa091 (Unity) GetStacktrace
0x00007ff7083b42ce (Unity) DebugStringToFile
0x00007ff705cd84f8 (Unity) DebugLogHandler_CUSTOM_Internal_Log
0x0000015234cbc9aa (Mono JIT Code) (wrapper managed-to-native) UnityEngine.DebugLogHandler:Internal_Log (UnityEngine.LogType,UnityEngine.LogOption,string,UnityEngine.Object)
0x0000015234cbc81b (Mono JIT Code) UnityEngine.DebugLogHandler:LogFormat (UnityEngine.LogType,UnityEngine.Object,string,object[])
0x000001527efa2d2e (Mono JIT Code) DCL.Diagnostics.DebugLogReportHandler:LogInternal (UnityEngine.LogType,DCL.Diagnostics.ReportData,UnityEngine.Object,object) (at D:/Decentraland/unity-explorer-ssh/Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/ReportsHandling/Handlers/DebugLogReportHandler.cs:73)
0x0000015234d2aeb6 (Mono JIT Code) DCL.Diagnostics.ReportHandlerBase:Log (UnityEngine.LogType,DCL.Diagnostics.ReportData,UnityEngine.Object,object) (at D:/Decentraland/unity-explorer-ssh/Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/ReportsHandling/Handlers/ReportHandlerBase.cs:28)
0x0000015234cba4a0 (Mono JIT Code) DCL.Diagnostics.ReportHubLogger:Log (UnityEngine.LogType,DCL.Diagnostics.ReportData,object,UnityEngine.Object,DCL.Diagnostics.ReportHandler) (at D:/Decentraland/unity-explorer-ssh/Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/ReportsHandling/ReportHubLogger.cs:73)
0x000001527efa2793 (Mono JIT Code) DCL.Diagnostics.ReportHub:LogError (DCL.Diagnostics.ReportData,object,DCL.Diagnostics.ReportHandler) (at D:/Decentraland/unity-explorer-ssh/Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/ReportsHandling/ReportHub.cs:53)
0x000001527efa1cc3 (Mono JIT Code) SceneRuntime.Apis.UnityOpsApi:Error (object) (at D:/Decentraland/unity-explorer-ssh/Explorer/Assets/Scripts/SceneRuntime/Apis/UnityOpsApi.cs:45)
0x00000154539c6f54 (Mono JIT Code) (wrapper runtime-invoke) <Module>:runtime_invoke_void__this___object (object,intptr,intptr,intptr)
0x00007fff73264c1e (mono-2.0-bdwgc) mono_jit_runtime_invoke (at C:/build/output/Unity-Technologies/mono/mono/mini/mini-runtime.c:3445)
0x00007fff7319d254 (mono-2.0-bdwgc) do_runtime_invoke (at C:/build/output/Unity-Technologies/mono/mono/metadata/object.c:3068)
0x00007fff731a4b5f (mono-2.0-bdwgc) mono_runtime_try_invoke_array (at C:/build/output/Unity-Technologies/mono/mono/metadata/object.c:5770)
0x00007fff730d4ee2 (mono-2.0-bdwgc) ves_icall_InternalInvoke (at C:/build/output/Unity-Technologies/mono/mono/metadata/icall.c:4041)
0x00007fff730fb0c1 (mono-2.0-bdwgc) ves_icall_InternalInvoke_raw (at C:/build/output/Unity-Technologies/mono/mono/metadata/icall-def.h:692)
0x00000150f44f91d2 (Mono JIT Code) (wrapper managed-to-native) System.Reflection.RuntimeMethodInfo:InternalInvoke (System.Reflection.RuntimeMethodInfo,object,object[],System.Exception&)
0x00000150f44f6e03 (Mono JIT Code) System.Reflection.RuntimeMethodInfo:Invoke (object,System.Reflection.BindingFlags,System.Reflection.Binder,object[],System.Globalization.CultureInfo)
0x00000150f44f686f (Mono JIT Code) System.Reflection.MethodBase:Invoke (object,object[])
0x00000154539c12cb (Mono JIT Code) Microsoft.ClearScript.Util.InvokeHelpers/<>c:<InvokeMethod>b__0_0 (System.Reflection.MethodInfo,object,object[])
0x00000154539bf574 (Mono JIT Code) Microsoft.ClearScript.Util.InvokeHelpers:InvokeMethodInternal<T_REF> (Microsoft.ClearScript.Util.IHostInvokeContext,T_REF,object,object[],System.Func`4<T_REF, object, object[], object>,System.Type,Microsoft.ClearScript.ScriptMemberFlags)

```

### LINKED ISSUE #1383: [QA] Broken gameplay and no NPC animation on Teleperformance
### **Explorer Alpha build version:**
v0.11.0

### **Issue Description:**

The climbing wall game can be started, however, there is an asset on the floor that lifts up the player and it looks like the NPC is starting the climb through the floor up. 
Said NPC is in a static position, not performing the correct movements of the climbing.

### **Steps to reproduce:**

  1. Start the climbing wall game selecting an NPC
  2. Use the W to climb untill the NPC is visible

### **Expected behaviour:**
The game must be played normally, with the NPC visible as soon as it starts

### **Current behaviour:**
The NPC appears through the floor when mantaining the W pressed. There is no movement on it, it stays on the same position (has no animation). 

#### **Evidence:**

https://github.com/decentraland/unity-explorer/assets/139157886/9edcc9af-d2c3-47bc-ba47-a54af902da2d

### PR DESCRIPTION
**Pull request by lorux0**
## What does this PR change?

Fixes #1383
Fixes #1372

Two issues were identified:

1. The scenes expect the WebSocket API to return `data` as a string when the `type:text` in response to the `receive()` call. This particular scene, a `JSON.parse(data)` was being performed on the WebSocket response.
2. The climbing character had multiple states, but when set to `playing: true`, the `weight:0`, causing the animation to be invisible.

It is stated that:

> If the weight is explicitly set to 0 then the animation would not be viewed.

So if the animation is not visible then is the expected thing to do. Seems like unity-renderer does not consider the weight for the bone blending, but on the current animation system in unity-explorer, this works as it should be.

In conclusion the scene should fix the `weight` parameter.

Screenshot 2024-07-12 at 10 04 25

## How to test the changes?

1. Clear AB cache
2. Follow issue steps

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by mikhail-dcl — APPROVED** [review]

**Review by sandrade-dcl — APPROVED** [review]

**Review by anicalbano — APPROVED** [review]
🟢 QA reviewed
